### ISSUE TITLE
We need guns

### ISSUE THREAD

**Issue by YarikIV** [mekanism/Mekanism-Feature-Requests#111]
**Describe the the feature you'd like**
I think we need some sort of gun, like a rail or gauss gun for the mod, burning everything is quite fun but I need something better than the electric bow.

**Describe alternatives you've considered**
Maybe an upgrade for the mekatool that allows it to shoot iron/steel/osmium nuggets using charge as propellant. 

**Additional context**
,more dakka please.

**Comment by Oct0bass** [comment]
I don't think it should be a "gun" like modern guns exactly, but I do think mek needs a ranged weapon.

**Comment by YarikIV** [comment]
> I don't think it should be a "gun" like modern guns exactly, but I do think mek needs a ranged weapon.

Exactly. thats why I suggested a gauss gun upgrade for the mekatool or at least integrate the flamethrower to it.
